website/docs/r/private_endpoint_interface_link.html.markdown

```diff
@@ -10,6 +10,8 @@ description: |-
 
 `mongodbatlas_private_endpoint_interface_link` provides a Private Endpoint Interface Link resource. This represents a Private Endpoint Interface Link, which adds one interface endpoint to a private endpoint connection in an Atlas project.
 
+!> **WARNING:** This resource is deprecated and will be removed in the next major version
```

Comment:
Same as above:
Same as above..
Can we add:
"This resource is deprecated and will be removed in the next major version
Please transition to privatelink_endpoint_service as soon as possible "
And link to the privatelink_endpoint_service doc?